src/content/migrate/4.md

```diff
@@ -91,6 +100,8 @@ The `CommonsChunkPlugin` was removed. Instead the [`optimization.splitChunks`](/
 
 See documentation of the [`optimization.splitChunks`](/configuration/optimization/#optimization-splitchunks) for more details. The default configuration may already suit your needs.
 
+More information on this change is mentioned in [here](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693).
```

Comment:
Another one for additional reading section i would say

Reply:
noted.